packages/firestore/src/core/component_provider.ts

```diff
@@ -77,6 +78,7 @@ export interface ComponentConfiguration {
   clientId: ClientId;
   initialUser: User;
   maxConcurrentLimboResolutions: number;
+  timeToFirstByte?:TimeToFirstByteCallback,
```

Comment:
I think your `prettier` commit hook is not firing. There should be a space after the `:` and a semicolon instead of the comma.

Reply:
It's supposed to run as a pre-commit hook. It would suggest merging master and running `yarn` at the repo root.